src/thread.js

```diff
@@ -75,19 +140,24 @@ class Thread {
       }
     })
     this._db.events.on('write', (dbname, entry) => {
-      let post = entry.payload.value
-      post.postId = entry.hash
-      newPostFn(post)
+      if (entry.payload.op === 'ADD') {
+        let post = entry.payload.value
+        post.postId = entry.hash
+        newPostFn(post)
+      }
     })
   }
 
   async _load (odbAddress) {
     // TODO - threads should use the space keyring once pairwise DIDs are implemented
-    const dbKey = this._3id._mainKeyring.getDBKey()
-    const key = await this._orbitdb.keystore.importPrivateKey(dbKey)
-    this._db = await this._orbitdb.log(odbAddress || this._name, {
-      key,
-      write: ['*']
+    const identity = await this._3id._mainKeyring.getIdentity()
+    this._db = await this._orbitdb.feed(odbAddress || this._name, {
+      identity,
+      accessController: {
+        type: 'thread-access',
+        address: odbAddress,
```

Comment:
Wait, does this mean that the odbAddress for the accessController is the same as for the thread itself? I thought there would be one feed for the AC and one for the thread itself?
Am I missing something?

Reply:
yeah there is one feed for AC, its created AC, also different since different AC, but cleaned up naming friday (still up for discussion), just pushed up those changes this morning

Reply:
Oh, now I get what's going on I think 🙂